CMSgov/mdct-mcr
# MDCT 397: Fix avscan

## Description
[MDCT 397](https://qmacbis.atlassian.net/browse/MDCT-397)

### How to test
- Log into MCR dev AWS
- Go to `uploads-fix-scan-attachments-446712541566` bucket in S3
- Drag a file in and upload it
- Navigate to CloudWatch and find the avscan lambda logs `/aws/lambda/uploads-fix-scan-avScan`
- Find the log associated with your upload (should be most recent)
- Confirm scan passed 🎉 

For additional confirmation, open Cloudshell and enter this command
```
aws s3api get-object-tagging --bucket uploads-fix-scan-attachments-446712541566 --key {YOUR_FILE_NAME}
```
See if the tag comes back CLEAN

### Changed Dependencies
None

## Code author checklist

## Reviewer checklist (two different people)
